staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/union_content.go

```diff
@@ -48,7 +48,9 @@ func (c unionCAContent) Name() string {
 func (c unionCAContent) CurrentCABundleContent() []byte {
 	caBundles := [][]byte{}
 	for _, curr := range c {
-		caBundles = append(caBundles, curr.CurrentCABundleContent())
+		if currCABytes := curr.CurrentCABundleContent(); len(currCABytes) > 0 {
+			caBundles = append(caBundles, []byte(strings.TrimSpace(string(currCABytes))))
```

Comment:
what was the purpose of this change? this drops a trailing newline from the last cert, which seems like it can cause [problems for some loaders](https://www.google.com/search?q=pem+trailing+newline) (not sure about the golang loader)